CHANGELOG.md

```diff
@@ -20,25 +20,191 @@ We’ve updated the header component to support account information and links. A
+
+For example:
+
+```scss
+.app-width-container--wide {
+  @include nhsuk-width-container(1200px);
+}
+```
+
+You can use the generated classes to set the width of:
+
+- template container
+- header container
+- footer container
+
+It was already possible to set the page app width with the `$nhsuk-page-width` variable. This new feature is useful when creating additional custom page width classes.
+
+This change was introduced in [#1412: Allow creating custom width containers](https://github.com/nhsuk/nhsuk-frontend/pull/1412)
+
 :boom: **Breaking changes**
 
 You must make the following changes when you migrate to this release, or your service might break.
 
+#### Update file paths
+
+To make sure NHS.UK frontend's files do not conflict with your code, we've moved our package files from `packages` to `dist/nhsuk`.
+
+##### If you’re using Sass
+
+Replace `packages/` with `dist/nhsuk` for any `@forward`, `@use` or `@import` paths in your [Sass](https://sass-lang.com/) files.
+
+Before:
+
+```scss
+@forward "node_modules/nhsuk-frontend/packages/core";
+```
+
+After:
+
+```scss
+@forward "nhsuk-frontend/dist/nhsuk/core";
+```
+
+You must add `node_modules` to Sass load paths, by either:
+
+- calling the Sass compiler from the command line with the `--load-path node_modules` flag
+- using the JavaScript API with `loadPaths: ['node_modules']` in the `options` object
+
+##### If you’re using CSS
+
+For precompiled stylesheets, note the following path changes:
+
+- Copy or serve `node_modules/dist/nhsuk/nhsuk-frontend.min.css`, not the previous `node_modules/dist/nhsuk.min.css` stylesheet
+- Extract `nhsuk-frontend-<VERSION-NUMBER>.min.css` from the GitHub release zip file, not the previous `css/nhsuk-<VERSION-NUMBER>.min.css` stylesheet
+
+##### If you’re using JavaScript
+
+For JavaScript imported using a bundler, consolidate all `import` or `require()` calls to `nhsuk-frontend/packages/components/*` into a single statement:
+
+Before:
+
+```mjs
+import initButtons from 'nhsuk-frontend/packages/components/button/button.js'
+import initCheckboxes from 'nhsuk-frontend/packages/components/checkboxes/checkboxes.js'
+
+// Initialise all button components
+initButtons();
+
+// Initialise all radios components
+initRadios();
+```
+
+After:
+
+```mjs
+import { initButtons, initCheckboxes } from 'nhsuk-frontend'
+
+// Initialise all button components
+initButtons();
+
+// Initialise all radios components
+initRadios();
+```
+
+Or alternatively, you can initialise individual components only:
+
+```js
+import { Button, Checkboxes } from 'nhsuk-frontend';
+
+const $button = document.querySelector('.app-button')
+const $checkboxes = document.querySelector('.app-checkboxes')
+
+// Initialise single button component
+new Button($button);
+
+// Initialise single checkboxes component
+new Checkboxes($checkboxes);
+```
+
+For precompiled JavaScript, note the following path changes:
```

Comment:
Swap the order of these so that it comes before the bundler instructions maybe? (I suspect using the precompiled JavaScript is way more common?)